components/cosmetic_filters/resources/data/element_picker.html

```diff
@@ -259,6 +279,13 @@
     position: absolute;
   }
 
+  .drag-indicator svg {
+    position: static;
+  }
+  .drag-indicator svg path{
+    fill: var(--primary);
+  }
```

Comment:
is this `drag-indicator` still draggable? If not, maybe lets rename it to collapse-button or something